app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessagePresenter.kt

```diff
@@ -32,12 +32,17 @@ class MessagePresenter {
         gattClient.stopClient()
     }
 
-    fun convertToPreview(data: String): ArrayList<String> {
+    fun convertToPreview(data: String): Pair<Boolean, ArrayList<String>> {
+        var flag = true
```

Comment:
When will flag be true/false

Reply:
If the entire message contains valid characters, it will be true, else false.

Reply:
Then change name to valid, same above

Reply:
Done.